internal/getproviders/multi_source_test.go

```diff
@@ -160,7 +160,7 @@ func TestMultiSourceAvailableVersions(t *testing.T) {
 			{Source: s2},
 		}
 
-		_, err := multi.AvailableVersions(addrs.NewDefaultProvider("foo"))
+		_, _, err := multi.AvailableVersions(addrs.NewDefaultProvider("foo"))
```

Comment:
Same testing thought as the memoize source. I don't have any good suggestions for adding tests here, so it may be more work than it's worth…

Reply:
I'll see what I can do!

I am also going to add an e2e test - the registry folks are figuring out what provider we can publish with a warning (possibly the "scaffolding" template). That'll exercise multisource more thoroughly.